l10n_be_vat_reports/__manifest__.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+#
+#     This file is part of l10n_be_mis_reports,
+#     an Odoo module.
```

Comment:
This comment is incorrect. BTW, you could simplify the copyright headers using the OCA template and also add Odoo and CoopIT Easy ?